openlibrary/macros/WorkCarousel.html

```diff
@@ -0,0 +1,5 @@
+$def with(work)
+
+$:render_template("home/custom_ia_carousel", title="You may also like:", key="related-subjects-carousel", work_id=work.key, _type="subjects", limit=42)
```

Comment:
If it's not too difficult, I liked having the subject name in the title. It made it clearer as to why these works were being recommended. (i.e. "Other works in Fiction")